.github/workflows/ci.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         run: psql --echo-errors --quiet -c '\timing off' -c "CREATE DATABASE ${TEST_DATABASE_NAME};" ${ADMIN_DATABASE_URL}
 
       - name: river migrate-up
-        run: river migrate-up --database-url "$TEST_DATABASE_URL"
+        run: river migrate-up --database-url "$TEST_DATABASE_URL" --max-steps 5 # temporarily include max steps so tests can pass with unique fixes
```

Comment:
Change needed for tests to pass, but should be reverted in https://github.com/riverqueue/riverqueue-python/pull/38.